rsyslog-logger.go

```diff
@@ -21,18 +22,16 @@ const RsyslogTemplate = `
 # File - {{ .FileName }}
 ######################################
 
-module(load="imfile")
-
 input(type="imfile"
-			File="{{ .CWD }}/{{ .FileName }}"
-			Tag="{{ .CleanAppName }}	{{.TaskID}}"
-			statefile="{{ .TaskID }}"
-      Severity="info")
+		File="{{ .WorkDir }}/{{ .FileName }}"
```

Comment:
@aayushKumarJarvis We always seem to have some space vs tab issue in here. Can you please fix this?